pombola/south_africa/views/geolocalization.py

```diff
@@ -54,7 +54,8 @@ class LocationSearchForm(SearchForm):
         widget=forms.TextInput(attrs={'placeholder': 'Your address'}))
 
 
-class LatLonDetailBaseView(BasePlaceDetailView):
+class LatLonDetailBaseView(BaseDetailView):
```

Comment:
Changed the parent of this class to it previous parent's parent, because `BasePlaceDetailView` was just making unnecessary database queries the results of which this page wasn't using.